tests/system/providers/microsoft/azure/example_s3_to_azure_blob.py

```diff
@@ -0,0 +1,106 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.s3 import (
+    S3CreateBucketOperator,
+    S3CreateObjectOperator,
+    S3DeleteBucketOperator,
+    S3DeleteObjectsOperator,
+)
+from airflow.providers.microsoft.azure.transfers.s3_to_azure_blob import S3ToAzureBlobOperator
+from airflow.utils.trigger_rule import TriggerRule
+from tests.system.providers.amazon.aws.utils import SystemTestContextBuilder
+
+BLOB_NAME = os.environ.get("AZURE_BLOB_NAME", "file.txt")
+AZURE_CONTAINER_NAME = os.environ.get("AZURE_CONTAINER_NAME", "airflow")
+DAG_ID = "example_s3_to_azure_blob"
+PREFIX_NAME = os.environ.get("AZURE_PREFIX_NAME", "20230421")
+
+sys_test_context_task = SystemTestContextBuilder().build()
+SAMPLE_DATA = "some data"
+
+with DAG(
+    DAG_ID,
+    schedule=None,

```

Comment:
All system tests need to have this

```suggestion
    schedule="@once",
```